app/helpers/messages_helper.rb

```diff
@@ -0,0 +1,12 @@
+module MessagesHelper
+
+def reviewee_or_reviewer(message)
```

Comment:
@yangsong8  I have fixed this and changed the code to display Author instead of Reviewer in the chat box.